tcga/src/services/BigQuery.js

```diff
@@ -79,10 +87,19 @@ class BigQuery {
     dataQuery = dataQuery.where(whereClause);
 
     // Only now do we clone the count query before adding possible limits and offsets
-    const countQuery = dataQuery
-      .clone()
-      .countDistinct(`${tableAlias}.${this.primaryKey} as count`)
-      .toString();
+    let countQuery;
+    if (concatSelection) {
```

Comment:
Think it would probably be better here if we just used the distinct parameter and removed concatSelection. Concat works with 1 or more fields and it is only being used for the count query which may bring confusion when using the function.